gradle-sls-packaging/src/main/java/com/palantir/gradle/dist/service/tasks/LaunchConfig.java

```diff
@@ -168,9 +168,10 @@ static void action(Params params) {
                         .addAllJvmOpts(avxOptions)
                         .addAllJvmOpts(params.getAddJava8GcLogging().get() ? java8gcLoggingOptions : ImmutableList.of())
                         // Java 11.0.16 introduced a potential memory leak issues when using the C2
-                        // compiler
+                        // compiler, resolved in 11.0.16.1
```

Comment:
[JDK-8291665](https://bugs.openjdk.org/browse/JDK-8291665) for future reference if needed